frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+                  >
+                    <CopyIcon className="w-4 h-4 mr-2" />
+                    Dangerously Copy
+                  </Button>
+                </div>
+                <div className="flex items-center mt-6 text-sm">
+                  <Checkbox
+                    id="backup"
+                    required
+                    onCheckedChange={() => setIsBackedUp(!backedUp)}
+                  />
+                  <Label htmlFor="backup" className="ml-2">
+                    I've backed up my recovery phrase to my wallet in a private
+                    and secure place
+                  </Label>
+                </div>
+                {backedUp && !info?.albyAccountConnected && (
+                  <div className="flex text-sm">
+                    <Checkbox
+                      id="backup2"
+                      required
+                      onCheckedChange={() => setIsBackedUp2(!backedUp2)}
+                    />
+                    <Label
+                      htmlFor="backup2"
+                      className="ml-2 text-sm text-foreground"
+                    >
+                      I understand the recovery phrase AND a backup of my hub
+                      data directory is required to recover funds from my
+                      lightning channels.
+                    </Label>
+                  </div>
+                )}
+                <div className="flex justify-end gap-2 items-center">
+                  <div className="flex justify-center">
+                    <Button type="submit" size="lg">
+                      Finish Backup
+                    </Button>
+                  </div>
+                </div>
+              </form>
+            </DialogContent>
+          </Dialog>
+        </div>
+      )}
+      <>
+        <Separator className="my-6" />
+        <div className="flex flex-col gap-8">
+          <div>
+            <h3 className="text-lg font-medium">Channels Backup</h3>
+            <p className="text-sm text-muted-foreground">
+              Your spending balance is stored in your lightning channels. In
+              case of recovery or migration or your Alby Hub, they need to be
+              backed up every time you open a new channel.
+            </p>
+          </div>
+
+          <div>
+            <div className="flex gap-2 mb-1 items-center">
+              <h3 className="text-sm font-medium">Automated Channels Backup</h3>
+              {info?.albyAccountConnected ? (
+                <Badge variant={"positive"}>Active</Badge>
+              ) : (
+                <Badge>Recommended</Badge>
+              )}
+            </div>
+            {info?.albyAccountConnected ? (
+              <>
+                <p className="text-muted-foreground text-sm mb-8">
+                  Your channel state is backed up automatically after each
+                  channels creation. Potential recovery will trigger channel
```

Comment:
```suggestion
                  channels creation. Using an external recovery tool and your recovery phrase, you can request channel
```

Reply:
i didn't understand this fully. what is external recovery tool here and how this is fitting in existing sentences

Reply:
https://github.com/getAlby/hub-recovery

Reply:
resolved